Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 4
+version = 3
```

Comment:
is this a symptom of running `nix build` rather than `cargo build` here?

Reply:
Yeah, I just deleted it and retried again and it still is version 3. I'm on rust 1.83, so it should be correct